portal/server/helpers/pageSpecificValidationErrors.test.js

```diff
@@ -250,6 +250,58 @@ describe('page specific validation errors', () => {
       expect(result).toEqual(expected);
     });
 
+    it('displays validation errors when user skips mandatory fields', () => {
```

Comment:
formatting

also companies house is optional isnt it?

Reply:
Done, yes